### PR TITLE
Task-6

### PR DESCRIPTION
**Pull request by ngnaoh**
https://edu-redmine.sun-asterisk.vn/issues/35224

### CODE REVIEW

**Comment by ngnaoh** [comment]
ready

**Comment by manhnt-0960** [comment]
conflicted

**Comment by ngnaoh** [comment]
ready

File: src/scss/components/_new.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    .date {
+      padding: 12px 0 12px 4px;
+      font-style: italic;
+      color: var(--grey-200);
+      font-weight: 400;
+      font-size: 12px;
+    }
+    .title {
+      font-size: 14px;
+      color: var(--green-100);
```
Comment (manhnt-0960) [inline src/scss/components/_new.scss:32]: dùng $ luôn e

**Pull request merged**